src/al.jl

```diff
@@ -76,24 +84,24 @@ function al(nlp :: AbstractNLPModel; max_iter :: Int = 1000, max_time :: Real =
 
   while !(solved || tired)
     # solve subproblem
-    S = with_logger(NullLogger()) do
+    S = with_logger(subsolver_logger) do
       tron(al_nlp, x = copy(al_nlp.x))
     end
 
     normcx = norm(al_nlp.cx)
 
     if normcx <= eta
       update_y!(al_nlp)
-      eta = T.(eta / (al_nlp.mu)^0.9)
+      eta = eta / al_nlp.mu^T(0.9)
```

Comment:
Greek letters?

Reply:
I want to make the complete change to greek on a separate PR